go/roothash/api/block/header.go

```diff
@@ -17,6 +19,16 @@ var ErrInvalidVersion = errors.New("roothash: invalid version")
 // HeaderType is the type of header.
 type HeaderType uint8
 
+// TimestampType is a custom time stamp type that encodes like time.Time when
+// marshaling to JSON.
+type TimestampType uint64
```

Comment:
This should just be `Timestamp`. The `HeaderType` above has `Type` because it's the "header type" ;)

Reply:
Oh, ok :slightly_smiling_face:. Fixed.